charts/fint-altinn-ebevis-service/values.yaml

```diff
@@ -3,6 +3,7 @@
 # Declare variables to be passed into your templates.
 
 nameOverride: "altinn-ebevis-service"
+xmx: 644 
```

Comment:
is this the default value we will set for all applications?

Reply:
No. This is for "altinn-ebevis-service"